### PR TITLE
Rename sensor to imager

### PR DESCRIPTION
**Pull request by npentrel**
Am I misunderstanding this or are we using sensor as a synonym

### CODE REVIEW

**Review by njooma — APPROVED** [review]
This is fine by me but this text was copied from RDK so we should change there as well cc @bazile-clyde

**Comment by bazile-clyde** [comment]
@npentrel I didn't write this but I suspect the reason the word "camera" wasn't used is because this method was implemented for the [Intel Real Sense](https://www.intelrealsense.com/depth-camera-d435/). 

That is one camera with multiple "imagers" as they call it. Each takes a separate image and we use the timestamps to confirm both images were taken at roughly the same time. 

To answer your question, no, "sensor" was not used synonymously with "camera". However, I'm not opposed to changing the word here, though, "camera" might be a bit overloaded since now it's referring to the imagers and the camera housing the imagers, the camera component.

**Comment by npentrel** [comment]
should we use imgers then?

**Review by cheukt — APPROVED** [review]

**Comment by njooma** [comment]
@npentrel has there been any movement on this? I fear it becoming stale...

**Comment by npentrel** [comment]
let me find the pr on the rdk and ask there - and then I'll follow up here

**Comment by bazile-clyde** [comment]
> Ok - @bhaney added the RDK code - what do you think Bijan - should we change this to imagers?

@npentrel He's OOO for about two weeks, and my team owns this code. I'm okay with leaving it as is or changing it to "imagers."

**Comment by npentrel** [comment]
ok let's do imagers then

File: src/viam/components/camera/camera.py [hunk 1]
```diff
@@ -52,8 +52,8 @@ async def get_image(self, mime_type: str = "", *, timeout: Optional[float] = Non
 
     @abc.abstractmethod
     async def get_images(self, *, timeout: Optional[float] = None, **kwargs) -> Tuple[List[NamedImage], ResponseMetadata]:
-        """Get simultaneous images from different sensors, along with associated metadata.
-        This should not be used for getting a time series of images from the same sensor.
+        """Get simultaneous images from different cameras, along with associated metadata.
+        This should not be used for getting a time series of images from the same camera.
```
Comment (npentrel) [inline src/viam/components/camera/camera.py:56]:
```suggestion
        """Get simultaneous images from different imagers, along with associated metadata.
        This should not be used for getting a time series of images from the same imager.
```

**Comment by npentrel** [comment]
https://github.com/viamrobotics/rdk/pull/2833

**Review by njooma — APPROVED** [review]

**Review by bazile-clyde — APPROVED** [review]
LGTM!

**Pull request merged**